agent/service.go

```diff
@@ -538,8 +540,6 @@ func (ts *TaskService) Shutdown(ctx context.Context, req *taskAPI.ShutdownReques
 	log.G(ctx).WithFields(logrus.Fields{"id": req.ID, "now": req.Now}).Debug("shutdown")
 	ctx = namespaces.WithNamespace(ctx, defaultNamespace)
 
-	// We don't want to call runc.Shutdown here as it just os.Exits behind.
```

Comment:
How come these comments are removed?

Reply:
It's no longer true, containerd made some nice updates that got rid of any need to call `os.Exit` from `Shutdown`: https://github.com/containerd/containerd/commit/ae87730ad2511ae39239ac7aac75074768aa3c1b